ports/samd/samd_qspiflash.c

```diff
@@ -124,14 +124,6 @@ static void samd_peripherals_enable_cache(void) {
     CMCC->CTRL.bit.CEN = 1;
 }
 
-/*
-    @brief Run a single QSPI instruction.
-    @param command instruction code
-    @param iframe iframe register value (configured by caller according to command code)
-    @param addr the address to read or write from. If the instruction doesn't require an address, this parameter is meaningless.
-    @param buffer pointer to the data to be written or stored depending on the type is Read or Write
-    @param size the number of bytes to read or write.
-*/
```

Comment:
It's OK to leave the comments here, just maybe in a different format that matches existing code, eg:
```c
// Run a single QSPI instruction.
// Parameters are:
// - command: instruction code
// - iframe: ...
```

Reply:
Done